.github/workflows/upgrade-gradle-wrapper.yml

```diff
@@ -0,0 +1,24 @@
+name: 'Gradle Wrapper Upgrade'
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '0 8 * * *'
```

Comment:
What does this line mean?
How often would the cron run? 
Is it free for open-source repos?

Reply:
This action runs every day at 8 Am UTC.
Yes it is free for open source projects to run scheduled actions.
However it runs only on "maintained"/"active" repositories. In case there is no "action" happening in this repository for longer than 30 days, this action will stop.

You're free to schedule this like you want 🙃. Maybe once a week is also fine. Or every two weeks 🤷‍♂️ your call ☺

Reply:
Got it - thanks for the good explanation! 8 days sounds good for me 💯